Apps/WebClient/src/ClientApp/src/store/modules/user/mutations.ts

```diff
@@ -90,6 +91,12 @@ export const mutations: MutationTree<UserState> = {
         state.statusMessage = "success";
         state.status = LoadStatus.LOADED;
     },
+    setPatientData(state: UserState, patientData: PatientData) {
+        state.patientData = patientData;
+        state.error = false;
+        state.statusMessage = "success";
+        state.status = LoadStatus.PARTIALLY_LOADED;
```

Comment:
This needs a bit more logic: If user is loaded and patient is loaded then the status is loaded, otherwise partially_loaded. Same for setUserData

Reply:
@FuriousLlama I'm not sure why we want to merge these statuses. i.e. Why do we say LOADED when setUserPreference is completed and PARTIALLY_LOADED here? I think we need to discuss for this design as what we load and when and how do we handle different calls for user. Currently, I see all the other calls making LoadStatus to LOADED.
![image](https://user-images.githubusercontent.com/48332333/110369357-27706f00-7fff-11eb-9b46-322e51fda39b.png)

Reply:
We dont know which set of data will be loaded first, thats why the each mutator needs to check it before setting the store state. We will need to refactor the other stores as we go. The data stores we have (meds, lab, etc) all use this logic